src/app/modules/item/pages/item-display/item-display.component.ts

```diff
@@ -51,11 +52,12 @@ export class ItemDisplayComponent implements OnInit, AfterViewChecked, OnChanges
 
   ngOnInit(): void {
     if (!this.itemData) throw new Error('itemData must be set in ItemDisplayComponent');
-    this.taskService.configure(
-      this.itemData.route,
-      this.itemData.item.url ?? undefined,
-      this.itemData.route.attemptId ?? this.itemData.currentResult?.attemptId,
-    );
+    const url = this.itemData.item.url;
+    const attemptId = this.itemData.route.attemptId ?? this.itemData.currentResult?.attemptId;
```

Comment:
fixed

Reply:
Probably it would have been better to ensure "by design" that the input cannot have an url not set or no attempt (by working with something else than ItemData?). But maybe that complicates things for low benefits.

Reply:
I can do it using a `ItemDisplayComponent.selector: 'alg-item-display[url][attemptId][route]'`, as I remember you were puzzled about when to use this technique and when not to. So should I use this ?

Reply:
fixed